config/manager/env.yaml

```diff
@@ -17,3 +17,5 @@ spec:
                   fieldPath: metadata.namespace
             - name: KUBE_RBAC_PROXY_IMAGE
               value: "quay.io/openshift/origin-kube-rbac-proxy:latest"
+            - name: POLL_PERIOD_SECONDS
```

Comment:
this ds env var u don't need it here

Reply:
I thought in a previous comment you wanted to expose the stats poll people as a config option and the only existing way to do it is with an env var via controller since it generates the daemonset.

Reply:
so this patched to manager process what we expose here is what we render inside the controllers other DS env var stays in the ds manifest like the events log file for example and if someone wanted to change they chnage the manifset and the controller will generate new ds with the required envvar, make sense ?